obspy/core/trace.py

```diff
@@ -2779,6 +3241,207 @@ def _data_sanity_checks(value):
                "arrays are allowed for initialization.") % str(value.shape)
         raise ValueError(msg)
 
+class FrequencyDomainTrace(BaseTrace):
+
+    def ifft(self):
+        tr = Trace()
+        tr.stats = Stats()
+        tr.stats.starttime = self.stats.starttime
+        tr.data = fftpack.irfft(self.data)
+        return tr
+
+    def plot(self):
+        N = self.stats.npts
+        T = self.stats.delta
+        x = np.linspace(0.0, N*T, N)
+        yf = self.data
+        xf = np.linspace(0.0, 1.0/(2.0*T), N/2)
+        fig, ax = plt.subplots()
+        ax.plot(xf, 2.0/N * np.abs(yf[:N/2]))
```

Comment:
This is not really correct due to the above mentioned issue with the return type of scipy's rfft.